app/src/server.js

```diff
@@ -40,13 +40,15 @@ passport.use(
 passport.use(
   new SpidStrategy(
```

Comment:
for improved readability I would suggest to separate che creation of the strategy from the call to `passport.use`